pages/home/quickstart.mdx

```diff
@@ -3,28 +3,180 @@ title: "Start using Arcade"
+</Tabs.Tab>
+</Tabs>
+
+## Implement the tool call
+
+Create a new file, open it with your preferred editor and enter the code indicated in the next steps.
+
+<Tabs items={["Python", "JavaScript"]}> <Tabs.Tab>
+```python
+touch example_openai.py
+```
+</Tabs.Tab>
+<Tabs.Tab>
+```javascript
+touch example_openai.js
+```
+</Tabs.Tab>
+</Tabs>
+
+First, we need to instantiate the OpenAI client pointing to the Arcade's endpoint and using the Arcade API key you created prior.
+
+<Tabs items={["Python", "JavaScript"]}> <Tabs.Tab>
+
+```python
+from openai import OpenAI
+
+client = OpenAI(base_url="https://api.arcade.dev/v1", api_key="arcade_api_key")
+```
+
+</Tabs.Tab> <Tabs.Tab>
+
+```javascript
+import { OpenAI } from "openai";
+
+const client = new OpenAI({baseURL: "https://api.arcade.dev/v1", apiKey: "arcade_api_key"});
+```
+
+</Tabs.Tab> </Tabs>
+
+Next, we define the parameters for the request. In the `prompt` below, you can replace "your@email.com" with your own email address. The Gmail tool will send a message to yourself, which you'll be able to see in your inbox.
+
+<Tabs items={["Python", "JavaScript"]}> <Tabs.Tab>
+
+```python
+prompt = "send an email to your@email.com with the subject 'Meeting' and the body 'Let's meet at 3pm tomorrow'"
+tools = ["Google.SendEmail"]
+model = "gpt-4o"
+user_id = "your@email.com"
+```
+
+</Tabs.Tab> <Tabs.Tab>
+
+```typescript
+let prompt = "send an email to your@email.com with the subject 'Meeting' and the body 'Let's meet at 3pm tomorrow'"
+let tools = ["Google.SendEmail"]
+let model = "gpt-4o"
+let userId = "your@email.com"
+```
+
+</Tabs.Tab> </Tabs>
+
+<Note>
+In a real application, the `prompt` would come from your user and the `user_id` would be a unique identifier for your application's user (could be an email address, a UUID, etc.). In this example, use your own email address.
+</Note>
+
+And, finally, we send the request:
+
+<Tabs items={["Python", "JavaScript"]}> <Tabs.Tab>
+```python
+response = client.chat.completions.create(
+    model=model,
+    messages=[{"role": "user", "content": prompt}],
+    tools=tools,
+    tool_choice="generate",
+    user=user_id,
+)
+
+print(response.choices[0].message.content)
+```
+</Tabs.Tab>
+<Tabs.Tab>
+```typescript
+const response = await client.chat.completions.create({
+  model: model,
+  messages: [{ role: "user", content: prompt }],
+  tools: tools,
+  tool_choice: "generate",
+  user: userId,
+});
+
+console.log(response.data.choices[0].message.content);
+```
+</Tabs.Tab>
+</Tabs>
+
+## Execute the code
+
+<Tabs items={["Python", "JavaScript"]}>
+<Tabs.Tab>
+```python
+python3 example_openai.py
+```
+</Tabs.Tab>
+<Tabs.Tab>
+```javascript
+node example_openai.js
+```
+</Tabs.Tab>
+</Tabs>
+
+## Authorize the tool
+
+The first time you run this, you will see a URL to authorize the tool to send the message on your behalf:
+
+> Please go to this URL and authorize the action: https://accounts.google.com/o/oauth2/v2/auth?...
```

Comment:
I know I suggested the quote block `>` but I forgot that would end up showing the link as clickable. Maybe we do both:
```suggestion
> `Please go to this URL and authorize the action: https://accounts.google.com/o/oauth2/v2/auth?...`
```

Could you see if that looks ok rendered on the blog site?